README.md

```diff
@@ -86,4 +84,7 @@ Participation in the Argo CD project is governed by the [CNCF Code of Conduct](h
 1. [Using Argo CD & Datree for Stable Kubernetes CI/CD Deployments](https://youtu.be/17894DTru2Y)
 1. [How to create Argo CD Applications Automatically using ApplicationSet? "Automation of GitOps"](https://amralaayassen.medium.com/how-to-create-argocd-applications-automatically-using-applicationset-automation-of-the-gitops-59455eaf4f72)
 1. [Progressive Delivery with Service Mesh – Argo Rollouts with Istio](https://www.cncf.io/blog/2022/12/16/progressive-delivery-with-service-mesh-argo-rollouts-with-istio/)
-
+1. [Argo CD - A GitOps Tool for continuous Delivery](https://medium.com/@raosharadhi11/argo-cd-a-gitops-tool-for-continuous-delivery-12788d274516)
+1. [Argo CD — Resource Hooks (PreSync, PostSync, Sync, and SyncFail) and Sync Waves](https://medium.com/@raosharadhi11/argo-cd-resource-hooks-presync-postsync-sync-and-syncfail-and-sync-waves-e753c6f02d85)
+1. [Argo Notifications with Argo CD](https://medium.com/@raosharadhi11/argo-notifications-with-argo-cd-d6ca0dc54cbc)
+1. [ArgoCD with Vault using argocd-vault-plugin](https://medium.com/@raosharadhi11/argocd-with-vault-using-argocd-vault-plugin-dccbc302f0c2)
```

Comment:
```suggestion
1. [ArgoCD with Vault using argocd-vault-plugin](https://medium.com/@raosharadhi11/argocd-with-vault-using-argocd-vault-plugin-dccbc302f0c2)
```

I'm still not sure if we would like to link these blogs in the readme unless it's published on [argo blog medium](https://blog.argoproj.io/).